cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/UsingTest.scala

```diff
@@ -35,6 +35,18 @@ class UsingTest extends DocumentingTest {
       "CREATE INDEX FOR (p:Pioneer) ON (p.born)",
       "CREATE INDEX FOR (c:Country) ON (c.formed)",
       "CREATE INDEX FOR ()-[i:INVENTED_BY]-() ON (i.year)",
+      "CREATE INDEX FOR (n:Scientist) ON (n.name)",
+      "CREATE INDEX FOR (n:Science) ON (n.name)",
+      "CALL db.awaitIndexes",
+      """CREATE
+        |(liskov:Scientist {name: 'Liskov', born: 1939})-[:KNOWS]->(wing:Scientist {name: 'Wing', born: 1956})-[:RESEARCHED]->(cs:Science {name: 'Computer Science'})<-[:RESEARCHED]-(conway:Scientist {name: 'Conway', born: 1938}),
+        |(liskov)-[:RESEARCHED]->(cs),
+        |(wing)-[:RESEARCHED]->(:Science {name: 'Engineering'}),
+        |(chemistry:Science {name: 'Chemistry'})<-[:RESEARCHED]-(:Scientist {name: 'Curie', born: 1867}),
+        |(chemistry)<-[:RESEARCHED]-(:Scientist {name: 'Arden'}),
+        |(chemistry)<-[:RESEARCHED]-(:Scientist {name: 'Franklin'}),
+        |(chemistry)<-[:RESEARCHED]-(:Scientist {name: 'Harrison'})
+      """
```

Comment:
```suggestion
      "CALL db.awaitIndexes",
      """
```
Looks like the test changed a lot in 4.3, it now has completely different indexes and data. You still want just this one line after the index creation probably.